pkg/installer/template.go

```diff
@@ -20,7 +20,7 @@ type TemplateData struct {
 }
 
 func ExecuteTemplate(name, text string, data TemplateData) ([]byte, error) {
-	t, err := template.New(name).Funcs(sprig.TxtFuncMap()).Parse(text)
+	t, err := template.New(name).Funcs(sprig.TxtFuncMap()).Delims("{{{", "}}}").Parse(text)
```

Comment:
should we use `{{kurl }}` instead, or something else more specific than `{{{ }}}`?